main.cpp

```diff
@@ -367,6 +425,15 @@ void WashingMachine::displayInitState()
               << "Brand: " << brand << std::endl;
 }
 
+void WashingMachine::trackRPM(float RPM)
+{
+    while (RPM < rpm)
+        {
+            RPM += 1.5f;
+            std::cout << "Current RPM is " << RPM << std::endl;
+        }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise